skimage/data/__init__.py

```diff
@@ -12,13 +12,16 @@
 
 import numpy as _np
 
-from .. import data_dir
 from ..io import imread, use_plugin
 from .._shared._warnings import expected_warnings, warn
-from .. import img_as_bool
+from ..util.dtype import img_as_bool
```

Comment:
Try reverting this line. It might cause circular import.

Reply:
I think maybe the line that should be fixed is
```
    from .util.dtype import *
```
as we shouldn't be importing `*`

Reply:
That said, as is, I think the tests will pass, and as such, `scikit-image`'s functionality should be restored. I'm inclined on leaving this fix to Issue #3156 as a "good first issue"